.expeditor/end_to_end.pipeline.yml

```diff
@@ -53,6 +53,39 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
 
+  - label: "[:windows: hab-svc-load]"
+    command:
+      - powershell .expeditor/scripts/end_to_end/setup_environment.ps1 DEV test_supervisor_load_service
```

Comment:
*nit* maybe we should probably rename `setup_environment.ps1` since it's also executing the test now.